Makefile

```diff
@@ -26,6 +26,9 @@ REGISTRY ?= honeycombio
 # Which architecture to build - see $(ALL_ARCH) for options.
 ARCH ?= amd64
```

Comment:
I'd like to also build arm64 if possible :) but followup PR

Reply:
`ARCH=arm64 make` should do it.

Reply:
right, but we'll need to build docker images and include them in the release